package.json

```diff
@@ -10,8 +10,10 @@
     "node": ">= 14"
   },
   "scripts": {
+    "prestart": "yarn generate",
     "start": "start-storybook -p 6006",
-    "prebuild": "shx rm -rf packages/*/dist",
+    "generate": "esbuild scripts/generate.ts --bundle --platform=node --target=es2019 --format=cjs --external:theo | node",
```

Comment:
1. Does `--external:theo` make it run faster?
I'm pretty sure I'd work the same without it.

2. `generate` is kinda generic.
Is there a name that would convey the meaning a bit better?
Perhaps `tokens` or `themes` or even `generate:tokens`.

Reply:
Yes, there will also have to be a theme(s) generation - changed to `generate:tokens`.

Regarding `--external`, it's needed to overcome problems such as: https://github.com/evanw/esbuild/issues/247, otherwise Theo does not run with TypeScript.

Reply:
Ah I see. Theo itself is not a proper esm and can't be bundled by esbuild properly. Cool.